backend/game_server/lib/loops/loop_move.ex

```diff
@@ -0,0 +1,26 @@
+defmodule BernacleServer.Loops.LoopMove do
+	alias BernacleServer.Supervisors.CellSupervisor
+	alias BernacleServer.Loops.LoopsStateHolder
+
+  	def start_link do
+        {:ok, spawn_link(fn -> move_iteration(:os.system_time(:micro_seconds)) end)}
```

Comment:
have you think about using `Task` or other higher abstraction level of process spawning?